readinglist/tests/test_utils.py

```diff
@@ -36,3 +36,13 @@ def test_does_remove_middle_spaces(self):
 
     def test_idempotent_for_null_values(self):
         self.assertEqual(strip_whitespace(colander.null), colander.null)
+
+
+class ResourceIdTest(unittest.TestCase):
+    def test_string_resource_id(self):
+        self.assertEqual(resource_id('toto'), 'toto')
```

Comment:
I'm not sure about this dummy "string" resource.

I was thinking about "field unicity" implementation: the backend needs a handle on the schema/resource to obtain some details about fields (types, contraints, ...) to initialize the "storage constraints"

Let's think about it